Mathlib/SetTheory/Game/PGame.lean

```diff
@@ -301,26 +297,28 @@ macro "pgame_wf_tac" : tactic =>
 -- Register some consequences of pgame_wf_tac as simp-lemmas for convenience
 -- (which are applied by default for WF goals)
 
+variable {xl xr : Type u} {xR : xr → PGame}
```

Comment:
This seems a bit inconsistent; can you move `xR` to the  two lemmas that use it?